apps/condo/domains/notification/adapters/appleAdapter.js

```diff
@@ -216,26 +215,41 @@ class AppleAdapter {
         }
 
         // NOTE: we try to fire Apple push request only if Apple push was initialized and we have some real notifications
-        if (!isNull(this.#app) && !isEmpty(notifications)) {
-            try {
-                const appleResult = await this.#app.sendAll(notifications, isVoIP)
-
-                if (!isEmpty(appleResult.responses)) {
-                    appleResult.responses = appleResult.responses.map(
-                        (response, idx) =>
-                            ({
-                                ...response,
-                                pushToken: notifications[idx].token,
-                                pushType: pushTypes[notifications[idx].token],
-                            })
-                    )
+        if (!isNull(this.#config) && !isEmpty(notifications)) {
+            const notificationsSortedByAppId = {}
+            for (const notification of notifications) {
+                const appId = notification.appId
+                if (!notificationsSortedByAppId[appId]) notificationsSortedByAppId[appId] = [notification]
+                else notificationsSortedByAppId[appId].push(notification)
+            }
+            for (const appId of Object.keys(notificationsSortedByAppId)) {
+                const currentConfig = this.#config[appId]
+                if (!currentConfig) {
+                    logger.info({ msg: 'Unknown appId. Config was not found', appId })
+                    continue
+                }
+                const currentNotificationsBatch = notificationsSortedByAppId[appId]
+                const app = new AppleMessaging(currentConfig)
```

Comment:
Single connection will be implemented later?

Reply:
It's better to create this instance inside the constructor